### DISCUSSION TITLE
Tell me your favorite songs.

### DISCUSSION THREAD

**Discussion by TakumaMiwa** [NAIST-SE/AttractingContributors#861] [2023 Attraction Contributions]
I want to know your favorite songs.

https://github.com/TakumaMiwa/AttractingContributors-Anime/tree/main

**Comment by hyuga-na** [comment]
I added my favorite song.
Would you tell me that dishes you recommend?
 ↓
https://github.com/hyuga-na/What-dishes-do-you-recommend-.git
